Classes/Util.php

```diff
@@ -348,7 +348,9 @@ public static function initializeTsfe($pageId, $language = 0, $useCache = TRUE)
 			$GLOBALS['TSFE']->getConfigArray();
 
 			$GLOBALS['TSFE']->settingLanguage();
-			$GLOBALS['TSFE']->settingLocale();
+			if (!$useCache) {
```

Comment:
We don't need to execute settingLocale here if we use cache as it will be executed later anyway.